### PR TITLE
changed 4.0.91.aib to <version>.aib in url

### PR DESCRIPTION
**Pull request by prasad927**
Signed-off-by: prasad927 <Prasad.Chaudhari@progress.com>

### :nut_and_bolt: Description: What code changed, and why?
    Fix URL which are not working due to Bundle not being available now for download like:
    https://packages.chef.io/airgap_bundle/current/automate/4.0.91.aib 

### :chains: Related Resources
     https://chefio.atlassian.net/browse/SHIELD-141

### :white_check_mark: Checklist

**All PRs** must tick these:

- [X] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

**Review by atultherajput — APPROVED** [review]

File: components/docs-chef-io/content/automate/ha_aws_deploy_steps.md [hunk 1]
```diff
@@ -65,6 +65,10 @@ Run the following steps on Bastion Host Machine:
    "
    ```
 
+   {{< note >}} 
+     Chef Automate bundles are available for 60 days from the release of a version. However, the milestone release bundles are available for download forever.
```
Comment (atultherajput) [inline components/docs-chef-io/content/automate/ha_aws_deploy_steps.md:69]: Don't add space before Chef Automte.

File: components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md [hunk 2]
```diff
@@ -66,6 +66,10 @@ Follow the steps below to deploy Chef Automate High Availability (HA) on AWS (Am
    "
    ```
 
+   {{< note >}} 
+     Chef Automate bundles are available for 60 days from the release of a version. However, the milestone release bundles are available for download forever.
```
Comment (atultherajput) [inline components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md:70]: Don't add space before Chef Automte.

File: components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md [hunk 3]
```diff
@@ -71,6 +71,10 @@ sudo sed -i 's/SELINUX=enforcing/SELINUX=permissive/g' /etc/selinux/config
    "
    ```
 
+   {{< note >}} 
+     Chef Automate bundles are available for 60 days from the release of a version. However, the milestone release bundles are available for download forever.
```
Comment (atultherajput) [inline components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md:75]: Don't add space before Chef Automte.

File: components/docs-chef-io/content/automate/ha_upgrade_introduction.md [hunk 4]
```diff
@@ -28,11 +28,15 @@ Steps to upgrade the Chef Automate HA are as shown below:
...
   ```
-  Download specific version bundle with this, example version: 4.0.91:
+  Download specific version bundle with this:
   ```bash
-  curl https://packages.chef.io/airgap_bundle/current/automate/4.0.91.aib -o automate-4.0.91.aib
+  curl https://packages.chef.io/airgap_bundle/current/automate/<version>.aib -o automate-<version>.aib
   ```
 
+  {{< note >}} 
+    Chef Automate bundles are available for 60 days from the release of a version. However, the milestone release bundles are available for download forever.
```
Comment (atultherajput) [inline components/docs-chef-io/content/automate/ha_upgrade_introduction.md:37]: Don't add space before Chef Automte.

**Review by atultherajput — CHANGES_REQUESTED** [review]

File: components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md (hunk 2 above)
Comment (prasad927) [reply components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md]: updated

**Review by atultherajput — APPROVED** [review]

File: components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md (hunk 2 above)
Comment (prasad927) [reply components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md]: updated

File: components/docs-chef-io/content/automate/ha_upgrade_introduction.md (hunk 4 above)
Comment (prasad927) [reply components/docs-chef-io/content/automate/ha_upgrade_introduction.md]: updated

**Review by vivekshankar1 — APPROVED** [review]

**Pull request merged**